WPMedia/fusion-news-theme-blocks
# PEN-670 - Update card list to not have image links in tab sequence

## Description

Update to the card list block (missed in previous PR) to not have image links within the tab sequence

Related PR - https://github.com/WPMedia/fusion-news-theme-blocks/pull/725

## Jira Ticket
- [PEN-670](https://arcpublishing.atlassian.net/browse/PEN-670)

## Acceptance Criteria

Screen reader should read the available articles as heading links only. For instance, "Apex Legends’ Season 4 trailer provides few answers......" heading level 2 link

## Test Steps

1. Checkout this branch `git checkout PEN-670-card-list`
2. In Fusion repo run with `npx fusion start -f -l @wpmedia/card-list-block`
3. Navigate to test page - http://localhost/all-blocks/?_website=the-gazette
4. Using your keyboard tab though the page - focusing on the Card List block on the right hand side
5. Verify only the headlines are focused and not the image links

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
